src/redux/modules/auth/middleware.js

```diff
@@ -16,6 +16,9 @@ const authMiddleware = ({ dispatch }) => next => (action) => {
         } else {
           window.history.replaceState({}, document.title, '.');
           dispatch(loginSuccess(authResult));
+          // we can use "push" to control the path we want to redirect
```

Comment:
how are we planning on doing redirects?

Reply:
see below `dispatch(push('/foo'));` for example.

Reply:
But im not married to this method. if you have something better I am all for it :)

Reply:
where are the docs for this stuff???